File: patterns/javascript-patterns/pattern99.js
// Pattern 99: ...

//     *
//    * *
//   *   *
//  *     *
// *********

let n = 5;
let px = n;
let py = n;

for (let i = 1; i <= n; i++) {
    let row = '';

    for (let j = 1; j < n * 2; j++) {
        if (j === px || j === py || i === n) {
            row += '*';
        } else {
            row += ' ';
        }
    }

    px--;
    py++;
    console.log(row);
}

File: patterns/python-patterns/pattern99.py
# Pattern 99: ...

#     *
#    * *
#   *   *
#  *     *
# *********

n = 5
px = n
py = n

for i in range(1, n + 1):
    for j in range(1, n * 2):
        if j == px or j == py or i == n:
            print("*", end="")
        else:
            print(" ", end="")
    px -= 1
    py += 1
    print()
